mappings/mapper.go

```diff
@@ -51,9 +53,19 @@ func (r *RoleMapper) GetRoleMapping(IP string) (*RoleMappingResult, error) {
 		return nil, err
 	}
 
-	// Determine if normalized role is allowed to be used in pod's namespace
+	externalID, err := r.extractExternalID(pod)
```

Comment:
Are there any errors that could be returned? If not, this can be simplified

Reply:
Understood, not currently. I did it like this to be consistent with RoleMapper.extractRoleARN, I can strip out the error passing/checking to simplify.

Reply:
This has been simplified.